packages/client/src/OsdkObjectFrom.ts

```diff
@@ -19,8 +19,11 @@ import type {
   ObjectOrInterfaceDefinition,
   ObjectTypeDefinition,
 } from "@osdk/api";
-import type { OsdkBase, OsdkObjectPrimaryKeyType } from "@osdk/client.api";
-import type { OsdkObjectPropertyType } from "./Definitions.js";
+import type {
+  OsdkBase,
```

Comment:
I think this might be a good example of something we don't want in the surface per se. Should we just be exporting `Osdk<>` from `@osdk/client.api` instead? Does that remove the need to export some of these?

Non blocking comment of course, just something for us to look at. Could be now. Could be later.